lib/rules/jsx-sort-props.js

```diff
@@ -107,7 +107,7 @@ const generateFixerFunction = (node, context, reservedList) => {
       });
     });
 
-    fixers.sort((a, b) => a.range[0] < b.range[0]);
+    fixers.sort((a, b) => b.range[0] - a.range[0]);
```

Comment:
Woah great fix here. @tihonove could you explain how this fix works?

Reply:
Comparators are supposed to only return a negative, positive, or zero number - previously it returned a boolean here.